dashboard/test/ui/features/step_definitions/account_steps.rb

```diff
@@ -17,6 +17,7 @@
 end
 
 Given(/^I sign in as "([^"]*)"( and go home)?$/) do |name, home|
+  steps "And I wait for 3 seconds"
   navigate_to replace_hostname('http://studio.code.org/reset_session')
```

Comment:
If I'm understanding right, the problem is caused by the way `reset_session` interacts with whatever else this test run has in-flight. Would it make sense to, instead of adding an unexplained `wait` here, to wrap these two steps in their own function called `reset_session` or something similar?

In any case, I think we should comment why the wait is necessary.

Reply:
Great idea. I added a new function with a context comment